### PR TITLE
qbs: init at 1.8

### PR DESCRIPTION
**Pull request by expipiplus1**

### CODE REVIEW

**Review by 0xABAB — APPROVED** [review]

File: pkgs/development/tools/build-managers/qbs/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  };
+
+  enableParallelBuilding = true;
+
+  buildInputs = with qt5; [
+    qtbase
+    qtscript
+  ];
+
+  installFlags = [ "INSTALL_ROOT=$(out)" ];
```
Comment (0xABAB) [inline pkgs/development/tools/build-managers/qbs/default.nix]: Wrong type of parentheses?

**Review by 0xABAB — CHANGES_REQUESTED** [review]
Retracting my former OK.

File: pkgs/development/tools/build-managers/qbs/default.nix [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    qmake -r qbs.pro
+    make
+  '';
+
+  postInstall = ''
+    mv $out/usr/local/* "$out"
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Qbs is a tool that helps simplify the build process for developing projects across multiple platforms";
```
Comment (pSub) [inline pkgs/development/tools/build-managers/qbs/default.nix]: `description` should not contain the name of the package. See https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes.

File: pkgs/development/tools/build-managers/qbs/default.nix (hunk 1 above)
Comment (pSub) [inline pkgs/development/tools/build-managers/qbs/default.nix]: Looks good to me. Do you mean the inner or outer parentheses?

**Review by pSub — CHANGES_REQUESTED** [review]

File: pkgs/development/tools/build-managers/qbs/default.nix (hunk 1 above)
Comment (0xABAB) [inline pkgs/development/tools/build-managers/qbs/default.nix]: I meant those in `$(out)`. Shouldn't that be `${out}`? (Given that you see nothing wrong, I am probably just unaware of something.)

Comment (expipiplus1) [inline pkgs/development/tools/build-managers/qbs/default.nix]: This style is used in a few other places in nixpkgs

pkgs/applications/editors/focuswriter/default.nix
pkgs/applications/audio/iannix/default.nix
pkgs/applications/misc/cool-retro-term/default.nix

for example

File: pkgs/development/tools/build-managers/qbs/default.nix (hunk 2 above)
Comment (expipiplus1) [inline pkgs/development/tools/build-managers/qbs/default.nix]: Fixed

File: pkgs/development/tools/build-managers/qbs/default.nix (hunk 1 above)
Comment (expipiplus1) [inline pkgs/development/tools/build-managers/qbs/default.nix]: @0xABAB `${out}` would refer to the nix value `out` which doesn't exist. Here `$(out)` expands the `out` environment variable, present at build time.

Comment (pSub) [inline pkgs/development/tools/build-managers/qbs/default.nix]: @expipiplus1 yes that is correct.

**Comment by expipiplus1** [comment]
Thanks!

**Pull request merged**